start.sh

```diff
@@ -59,7 +49,7 @@ export VSO_AGENT_IGNORE=AZP_TOKEN,AZP_TOKEN_FILE
 print_header "1. Determining matching Azure Pipelines agent..."
 
 AZP_AGENT_PACKAGES=$(curl -LsS \
-    -u user:$(cat "$AZP_TOKEN_FILE") \
+    -u user:"$TOKEN" \
```

Comment:
This doesn't expose the token in the logs does it?

Reply:
Just double check and it does

Reply:
Replaced that line with the original line again and the token is no linger being exposed.  Tested in ss-ptlsbox